internal/config/image/cuda_image.go

```diff
@@ -111,24 +127,25 @@ func (i CUDA) GetRequirements() ([]string, error) {
 // HasDisableRequire checks for the value of the NVIDIA_DISABLE_REQUIRE. If set
 // to a valid (true) boolean value this can be used to disable the requirement checks
 func (i CUDA) HasDisableRequire() bool {
-	if disable, exists := i.env[EnvVarNvidiaDisableRequire]; exists {
-		// i.logger.Debugf("NVIDIA_DISABLE_REQUIRE=%v; skipping requirement checks", disable)
+	disable := i.Getenv(EnvVarNvidiaDisableRequire)
+	if disable != "" {
 		d, _ := strconv.ParseBool(disable)
 		return d
 	}
-
 	return false
 }
 
 // DevicesFromEnvvars returns the devices requested by the image through environment variables
 func (i CUDA) DevicesFromEnvvars(envVars ...string) VisibleDevices {
```

Comment:
Why the changes to this function? If we're going to clean it up, let's make sure that this is a separate commit or even PR.